modules/elements-color-picker/assets/scss/elemtents-color-picker.scss

```diff
@@ -0,0 +1,69 @@
+
+.elementor-element {
+
+	&-color-picker {
+		--swatch-size: 27px;
+		display: flex;
+		position: absolute;
+		width: calc( var( --count ) * var( --swatch-size ) );
+		height: var( --swatch-size );
+		top: 3px;
+		left: 3px;
+		border-radius: 3px;
+		opacity: 0;
+		pointer-events: none;
+		z-index: 1000;
+		border: 2px solid #FFF;
+		transition: opacity .3s, left .05s, top .05s, width .3s;
+
+		&[data-count='0'] {
```

Comment:
Explain.

Reply:
That was actually a stupid one. Removed it.